data/tribes/initialization/atlanteans/units.lua

```diff
@@ -197,6 +197,31 @@
 --
 --    **productionsite_workers_coming**: The string to display over this tribe's productionsites when more than 1 worker is coming.
 --
+--    **soldier_context**, **soldier_0_sg**, **soldier_0_pl**, **soldier_1_sg**, **soldier_1_pl**, **soldier_2_sg**, **soldier_2_pl**,
+--    **soldier_3_sg**, **soldier_3_pl**: The pgettext context for militarysites' soldier strings, and the corresponding **unlocalized**
```

Comment:
Please add some information that the strings must follow exactly the format below and only the name of the soldier can be defined for each tribe. Also add, that the NPGETTEXT entries need to start with the UNUSED_ part